### DISCUSSION TITLE
Detekt build fails due to IntelliJ bug?

### DISCUSSION THREAD

**Discussion by marschwar** [detekt/detekt#6082] [Q&A]
I keep getting the same message when building the detekt code base within IntelliJ.

```
Entry classpath.index is a duplicate but no duplicate handling strategy has been set. Please refer to 
https://docs.gradle.org/8.1.1/dsl/org.gradle.api.tasks.Copy.html#org.gradle.api.tasks.Copy:duplicatesStrategy for details.
```

I found that there is an [issue in the Jetbrains issue tracker](https://youtrack.jetbrains.com/issue/IDEA-305759/Gradle-cannot-handle-classpath.index-duplicates) for that already with a comment from @3flex . The issue is marked as resolved, but it clearly still exists. 

Is there anything we can do about it or is there a good workaround for it? On Linux I keep doing `find . -type f -name classpath.index | rm -rf`.

**Comment by 3flex** [comment]
Are you using IntelliJ 2023.1.1?

Also it seems like the problem is junit 4 brought in as a transitive dependency. If that's excluded then it will resolve for us. But if you're not using 2023.1.1 then please upgrade and see if it's resolved.

**Reply by marschwar** [reply]
~~The update seems to have done the trick.~~ Thank you

Edited: I still see this issue from time to time. I will try to find a stable way to reproduce.

**Reply by 3flex** [reply]
FYI someone else has replied on the IntelliJ issue a few days ago also saying they're also still seeing this.

**Reply by atulgpt** [reply]
> FYI someone else has replied on the IntelliJ issue a few days ago also saying they're also still seeing this.

I also faced this issue. My issue was coming from `report-(md|txt|html|sarif)` tasks. I deleted the build folder and reran which solved this issue for me

**Reply by 3flex** [reply]
Looks like it might actually be fixed in 2023.1.2, @atulgpt @marschwar have you seen it in those versions?

If it's still an issue this should be fixed upstream in IntelliJ.

**Reply by marschwar** [reply]
The problem just resurfaced in 2023.1.3 and 2023.1.4.

**Reply by 3flex** [reply]
I'm seeing this too. There's a new issue open: [IDEA-323268](https://youtrack.jetbrains.com/issue/IDEA-323268/IDEA-generates-classpath.index-files-when-running-the-debugger)
